doc/components.md

```diff
@@ -134,14 +136,15 @@ You can also change the current playing animation using the `updateAnimation` me
 
 For a working example, check this [source file](/doc/examples/flare/lib/main_component.dart).
 
-## Composed component
+## Composability of components
 
-A mixin that helps you to make a `Component` wraps other components. It is useful to group visual components through a hierarchy. When implemented, makes every item in its `components` collection field be updated and rendered with the same conditions.
+Sometimes it is useful to make your `Component` wrap other components. For example by grouping visual components through a hierarchy.
```

Comment:
we need to say here this is only available for BaseComponents

Reply:
Fixed!